src/commands/move-backend-to-organization.ts

```diff
@@ -0,0 +1,45 @@
+import {BaseCommand} from '../lib'
+import {getEnvironment} from '../lib/environments'
+import fetch from 'node-fetch'
+
+export default class MoveBackendToOrganization extends BaseCommand {
+  static description = 'Move a backend to an organization'
+
+  static examples = [
+    '$ slash-graphql move-backend-to-organization 0x123 0x1234',
```

Comment:
1 template and 1 example is better
 '$ slash-graphql move-backend-to-organization <what??> <what??>',